src/panels/config/integrations/integration-panels/ozw/ozw-config-node.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  CSSResultArray,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import memoizeOne from "memoize-one";
+import { navigate } from "../../../../../common/navigate";
+import { haStyle } from "../../../../../resources/styles";
+import type { HomeAssistant, Route } from "../../../../../types";
+import "../../../../../layouts/hass-tabs-subpage";
+import "./ozw-node-router";
+import { computeTail } from "./ozw-config-router";
+
+@customElement("ozw-config-node")
+class OZWConfigNode extends LitElement {
```

Comment:
Why do we have this element? All it does is show a different element? Why not load that one directly?

Reply:
Eventually there will be separate pages/tabs. Node status, node configuration, node associations, etc

Reply:
But that is taken care of by the `ozw-node-router` right? Not this component?